src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,29 @@
+import * as cn from 'classnames';
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+type sizeType = 's' | 'm' | 'l';
```

Comment:
для таких перечислений используем enum. Все типы и енамы именуем с заглавной буквы
```suggestion
enum SizeType = {
  s = 's',
  m = 'm',
  l = 'l',
}
```

Reply:
done